library/Makefile

```diff
@@ -1,8 +1,23 @@
 
 # Also see "include/mbedtls/config.h"
 
+ifeq ($(IAR_BUILD),1)
+CFLAGS	?= --arm -Om --silent
+
+# gcc comparable warning flags are not present in IAR.
+# All warnings are enabled by default.
+WARNING_CFLAGS ?=
+
+# IAR archive command does not have ar compatible options 
+ARFLAGS =
```

Comment:
Do you mean
```ARFLAGS ?=``` 
this means retain original value. The purpose here is to empty the default ```rv``` from this variable. Because ```iarchive``` does not like ```ar``` like options.